app/assets/javascripts/app/views.js

```diff
@@ -65,7 +65,8 @@ app.views.Base = Backbone.View.extend({
     var self = this;
     _.each(this.subviews, function(property, selector){
       var view = _.isFunction(self[property]) ? self[property]() : self[property];
-      if(view) {
+      if (view && self.$(selector).length > 0) {
+        self.$(selector).first().empty();
```

Comment:
Ok, so when we run render for a second time, in case of `app.views.AspectMembership` `selector` matches to `.aspect_membership_dropdown` twice: to the origianal placeholder and to the `div` element generated by a previous render call (which is inside the first placeholder `div`). Since `html(...)` applies to every matched element, it applies to both elements. DOM looks ok after that, but for some reason further `render` calls don't update it properly anymore. Maybe `html(...)` call to nested elements produces some kind of undefined behaviour.

![2016-07-14 15 59 57](https://cloud.githubusercontent.com/assets/10187586/16840243/43673620-49dc-11e6-8819-cdbe5336692f.png)


So while I can fix it by more precise specification for the selector, it worth leaving explicit erase for the case when selector is specified so resulting in nested elements matches. Having explicit erase would make the difference so the next `.html(...)` call would apply only to the top level matched elements (all nested are erased and therefore don't match anymore). I guess this way it is safer.